assets/src/blocks/SocialMediaCards/SocialMediaCards.js

```diff
@@ -74,52 +79,25 @@ export class SocialMediaCards extends Component {
       <Fragment>
         <div>
           <TextControl
-            label={__('Title', 'planet4-blocks-backend')}
-            placeholder={__('Enter title', 'planet4-blocks-backend')}
-            help={__('Optional', 'planet4-blocks-backend')}
+            label={__('Title', 'p4ge')}
```

Comment:
Hi @Inwerpsel , The `planet4-blocks-backend` is the right text-domain.

Reply:
Also for the frontend? If I search for occurrences of `'p4ge'` in `*.js` files I find over 200 results in the block files, while for only 27 occurrences of `'planet4-blocks-backend'`.

Reply:
For frontend, we have `planet4-blocks`. Yes, we have mix of text-domains in repo. We will fix the remaining in separate PR.

Reply:
And keep in mind, that "frontend" in this context is definend from a user perspective, not development. It's for the strings that the end users see on the frontend.